cli/cmd/install/install.go

```diff
@@ -382,10 +386,26 @@ opentelemetryOperator:
 	return helmValues
 }
 
+func appendTelemetryVariables(version, timescaledb string, otel bool) string {
```

Comment:
why is timescaledb a string and otel a bool? aren't these both bools?

Reply:
`otel` variable is passed directly to tobs cli as a flag, so it is bool from t beginning. `timescaledb` is read from helm values using helm client and then parsed. Since we are not exactly sure what is the content, it was easier to pass it down as string and not try to parse it to bool.